apps/pie-docs/src/engineers/guidelines/browser-support.md

```diff
@@ -12,41 +12,41 @@ eleventyComputed:
 
 ## Browser support
 
-The number of browsers available to our users is vast, especially for those using mobile devices. Below, we have defined a rating system that helps us to specify the level of browser support we aim to achieve and to clarify what level of testing is required.
+Users have access to hundreds of browsers, so we've come up with a rating system to measure the level of browser support we want to achieve and the level of testing needed.
```

Comment:
I'd argue we're not really _measuring_ the level of support, because it's something we've defined. Also, I think a bit more explanation helps this paragraph say what it's meaning to say.

```suggestion
Users have access to hundreds of browsers, so instead of assessing them all individually we've come up with a rating system which defines different levels of browser support and the testing requirements for each level. We've assigned the highest ratings to the browsers that are most commonly used by our customers.
```

Reply:
Good call, we can make this change. I'll update the desgins. Thanks!

Reply:
👍